eng/scripts/prerelease.mjs

```diff
@@ -90,6 +103,24 @@ async function addPrereleaseNumber(changeCounts, packages) {
         await writeFile(packageJsonPath, JSON.stringify(newManifest, null, 2));
     }
 }
+
+/**
+ * @returns Date in format "YYYY-MM-DD-hh-mm-ss"
+ */
+function getIsoTime() {
+    const date = new Date().toISOString();
```

Comment:
I think I'd prefer the `new Date()` followed by `date.GetYear()` and `date.GetMonth()` etc etc etc, but frankly that's basically the same thing as what you're doing with the sliced `toISOString`. /shrug